Fixed a number of issues, most notably the sorting by "cit/year" issue

acheng98 opened the pull request:
Fixed sorting by "cit/year" by placing creation of pandas column before sort. 
Fixed extra counting of two excess "\<div>" classes (skip first and last div, which are in "ccl_top" and "ccl_bot"; relevant data in "ccl_mid")
Made get_author and get_year more concise and accurate by using .find()
Caused selenium driver to quit after execution, if called

Also, changed all spaces to tabs (sorry, personal preference :P), and made the default column 'cit/year' to sort along (descending)

WittmannF commented:
Such a complete PR is very welcome! The only thing that sads me is your preference to tabs hahah my only question is: is there any update that I should make in the documentation after your PR?

Thanks!

acheng98 commented:
Hi Fernando - I just forked your code for a bit of paper gathering myself, and just cleaned up the code just a little bit, I didn't test the compatibility with your examples or other files. I don't think there's any update necessary to documentation if everything still works, other than the fact that I changed the default sort column to 'cit/year'. 

Also, I just realized/remembered - I made the output a xlsx file rather than a csv, just commenting out your pd.to_csv() code - feel free to change back that functionality. 

Hope I covered everything!
